target/linux/mediatek/dts/mt7981b-wavlink-wl-wn586x3.dts

```diff
@@ -0,0 +1,244 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+
+#include "mt7981.dtsi"
+
+/ {
+	model = "WAVLINK WL-WN586X3";
+	compatible = "wavlink,wl-wn586x3", "mediatek,mt7981";
+
+	aliases {
+		led-boot = &led_status_blue;
+		led-failsafe = &led_status_blue;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_blue;
+		serial0 = &uart0;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&pio 11 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_blue: led@0 {
```

Comment:
It makes more sense IMHO to use within the `led@X` notation something similar to an address as `X` rather than sequential numbers. You could use the GPIO pin number (e.g., `led@10` here).

Reply:
It should be sequential numbers.
ref: https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/leds/leds-gpio.yaml

Reply:
That's when you use `led-X`, not `led@X`.

Reply:
IMO, GPIO pin number is not an address and ```-``` is better.